demos/init-app.php

```diff
@@ -51,8 +51,13 @@
     throw new \atk4\ui\Exception('Database error: ' . $e->getMessage());
 }
 
-[$rootUrl, $relUrl] = preg_split('~(?<=/)(?=demos(/|\?|$))|\?~s', $_SERVER['REQUEST_URI'], 3);
-$demosUrl = $rootUrl . 'demos/';
+// Current sub directory of webserver - makes it independent where to run the demo from
+// basically we want to run it just from within the ui repo, but this adds a layer of
+// better user experience, when a user tries to copy it elsewhere and it works
+$currDir = substr(str_replace($_SERVER['DOCUMENT_ROOT'], "", dirname(__FILE__)), 1);
```

Comment:
I belive should be not supported, we should actually not distribute tests/demos in release at all for security reasons.

Reply:
I'm seeing it from the perspective as a user (I for myself joined atk4 a year ago). 
The learning curve is ridiculous steep as there's a big lack of documentation. Reading everything out of the code without having examples makes everything even more time consuming. Which is unnecessary imo, if we just 
1) offer more examples
2) write documentations
3) make it as easy as possible to get results (raise usability)

Currently you have to try out a lot. 
If you just spend hours or days of getting a demo running, users jump off again before they even were able to see a welcome screen...

Therefore I'd be interested in opinions of other team members first

Reply:
please post usecase (step by step) that needs this change

Reply:
I think this is a positive change and it does not break anything. Sound reasoning on moving things outside.

Reply:
> 
> 
> please post usecase (step by step) that needs this change

as written, I need usecase that needs this change, otherwise `str_replace($_SERVER['DOCUMENT_ROOT'], "", dirname(__FILE__)` seems super hacky to me and it will not work on even slighly badly configured servers!